src/Middleware/ConcurrencyLimiter/src/QueuePolicies/ResettableBooleanCompletionSource.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Threading.Tasks;
+using System.Threading.Tasks.Sources;
+
+namespace Microsoft.AspNetCore.ConcurrencyLimiter
+{
+    internal class ResettableBooleanCompletionSource : IValueTaskSource<bool>
+    {
+        ManualResetValueTaskSourceCore<bool> _mrvts;
```

Comment:
nit: _mrvts isn't a great name. Too much Hungarian notation.

Reply:
Ah but it isn't a prefix; its the whole name so it isn't Hungarian notation, technically ;)

It is an overly verbose type name though :-/

How about `_valueTaskSource`?

e.g. 

`ManualResetValueTaskSourceCore<bool> _valueTaskSource;`